vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -45,13 +46,15 @@ jwtAuthenticationConverter.setJwtGrantedAuthoritiesConverter(new JwtGrantedAutho
  ```
 6. Now modify  
    `src/main/java/com/mycompany/myapp/security/oauth2/JwtGrantedAuthorityConverter.java` to accept the clientId. Three changes are required.
-  * Remove the @Component annotation on the class definition:
- ```java
+  * Remove the @Component annotation on the class definition
+ ```java{1}
```

Comment:
I think for the most part, the instructions reads much clearer now, especially with the highlighted code and maybe I've become more familiar with it. I'm guessing those familiar with Java will not flounder as we did. But wondering can annotation refer to multiple lines? Annotation to me could be single or many lines and that made it hard to get a firm idea of what to implement or look for. But if annotation is commonly used to refer to single lines of code, then this is a moot point.

Reply:
@jyunmitch Annotation is a Java feature, indicated by a line beginning with @Something. I'm reasonably sure it cannot be multi-line, certainly 99% of annotations are one line even if the spec allows more. https://docs.oracle.com/javase/tutorial/java/annotations/basics.html